src/electron/renderer.tsx

```diff
@@ -204,10 +200,17 @@ Sender.receive(message => {
 			break;
 		}
 		case ServerMessageType.Delete: {
-			if (app.getActiveView() === Types.AlvaView.Pages) {
+			if (
+				app.getActiveView() === Types.AlvaView.PageDetail &&
```

Comment:
The "avctive view" concept should/could be removed entirely

Reply:
At the moment this distinguishes between splashscreen and detail page – or would you handle that differently?